jsk_arc2017_baxter/euslisp/lib/arc-interface.l

```diff
@@ -335,7 +336,9 @@
                   (- (/ (m->mm (send bin-box :dimensions :x)) 2.0))
                   0.0 (/ (m->mm (send bin-box :dimensions :z)) 2.0))
                 offset) :world)
-      (send bin-coords :rotate (* pi (/ coords-pitch 180)) :y)
+      (send bin-coords :rotate (deg2rad (aref rpy 1)) :y)
+      (send bin-coords :rotate (deg2rad (aref rpy 2)) :z)
+      (send bin-coords :rotate (deg2rad (aref rpy 0)) :x)
```

Comment:
So, we should
* change `:rpy` order to `yaw-pitch-roll`
* change `:rpy` unit to radian
* change rotating order to `yaw-pitch-roll`
ex).
```
(send bin-coords :rotate (aref rpy 0) :z :local)
(send bin-coords :rotate (aref rpy 1) :y :local)
(send bin-coords :rotate (aref rpy 2) :x :local)
```
* change `:move-arm-body->bin-overlook-pose` to follow `:rpy` change